isic/core/forms/search.py

```diff
@@ -9,7 +9,7 @@
 class ImageSearchForm(forms.Form):
     query = forms.CharField(
         required=False,
-        widget=forms.TextInput(
+        widget=forms.SearchInput(
```

Comment:
My editor isn't picking this up for some reason, so we should make sure it works.